packages/jaeger-ui/src/components/SearchTracePage/SearchResults/index.test.js

```diff
@@ -175,8 +175,11 @@ describe('<SearchResults>', () => {
 
       it('when click on DownloadResults then call download function', () => {
         const originalBlob = global.Blob;
-        global.Blob = function (text, options) {
-          return { text, options };
+        global.Blob = class {
+          constructor(text, options) {
+            this.text = text;
+            this.options = options;
+          }
```

Comment:
@yurishkuro From my local, it seems like what really brought the `unit-test` failure is:
<img width="1036" alt="Screenshot 2023-08-10 at 12 47 34 PM" src="https://github.com/jaegertracing/jaeger-ui/assets/18136486/e42fc316-9e43-4ca4-bfc1-70890e2c6a9e">

(Not sure why it was missed in the CI logs)

For this error above, I think I made a mistake to convert `Blob` to an arrow function that is not able to call by `new`. 
 
So I change such `global.Blob` to an "explicit" class. Assuming this can:
1. Still resolve the origin lint warning
2. Attempt to make it clear so satisfy calling by keyword `new`.

How do you think of this?